fluidBuild.config.cjs

```diff
@@ -55,13 +55,25 @@ module.exports = {
 			"api-extractor:esnext",
 		],
 		"api": {
-			dependsOn: ["api-extractor:commonjs", "api-extractor:esnext"],
+			dependsOn: [
+				"api-extractor:commonjs",
+				"api-extractor:esnext",
+				"build:rename-types",
+				"build:rewrite-type-imports",
+			],
 			script: false,
 		},
 		"api-extractor:commonjs": [...tscDependsOn, "tsc"],
 		"api-extractor:esnext": [...tscDependsOn, "api-extractor:commonjs", "build:esnext"],
-		"build:docs": [...tscDependsOn, "tsc"],
-		"ci:build:docs": [...tscDependsOn, "tsc"],
+		"build:rename-types": ["api-extractor:esnext"],
+		"build:rewrite-type-imports": ["build:rename-types"],
+		"build:docs": [...tscDependsOn, "tsc", "build:rename-types", "build:rewrite-type-imports"],
+		"ci:build:docs": [
+			...tscDependsOn,
```

Comment:
Do these tasks that depend directly on `tsc` also need to depend on `tscDependsOn`?

Reply:
No; I opened https://github.com/microsoft/FluidFramework/pull/18686 to address that.